test/tools/router_check/validation.proto

```diff
@@ -46,7 +46,7 @@ message ValidationInput {
 
   // This pseudo-header field includes the HTTP method.
   // The options are GET, PUT, or POST. When not specified the field defaults to GET.
-  string method = 4 [(validate.rules).string = {in: ["GET", "POST", "PUT", ""]}];
```

Comment:
I think method should still be required? WDYT?

Reply:
@mattklein123  Based on existing [documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/tools/router_check.html) method is optional
```
:method
(optional, string) The request method. If not specified, the default method is GET. The options are GET, PUT, or POST.
```

Reply:
IMO I would make it required. I think the tests should be very specific so they are easy to visually understand.

Reply:
agree with the part about making it very explicit for consistency. I will make the change right away.

Reply:
@mattklein123 I made the change to make `method` mandatory. I have put min_bytes=3 . I didn;t want to put the exhaustive list there, but all methods have a length of 3. Is it a good constraint to have?